experiments/ClimaEarth/components/land/climaland_integrated.jl

```diff
@@ -440,6 +442,38 @@ end
 Interfacer.step!(sim::ClimaLandSimulation, t) = Interfacer.step!(sim.integrator, t - sim.integrator.t, true)
 Interfacer.reinit!(sim::ClimaLandSimulation, t) = Interfacer.reinit!(sim.integrator, t)
 
+function FieldExchanger.update_sim!(sim::ClimaLandSimulation, csf, turbulent_fluxes, area_fraction)
+    # update fields for radiative transfer
+    Interfacer.update_field!(sim, Val(:diffuse_fraction), csf.diffuse_fraction)
+    Interfacer.update_field!(sim, Val(:sw_d), csf.SW_d)
+    Interfacer.update_field!(sim, Val(:lw_d), csf.LW_d)
+    Interfacer.update_field!(sim, Val(:cos_zenith_angle), csf.cos_zenith_angle)
```

Comment:
To me, `cos_zenith_angle` is something that should not be updated/exchanged because it is global property of the system that depends only on the current data. ClimaLand can compute this internally.

Reply:
I see your point, but I don't think it makes sense to have the computation of `cos_zenith_angle` in two places. It's duplicated code, and they could have different ways of computing it and end up with different values. I'd like to keep it like this for now, and we can discuss computing it in ClimaLand for a future PR

Reply:
The zenith angle is a physical parameter and we already have a single shared package to ensure that they are computed in the same way (Insolation.jl). 

It will have to be computed internally anyway because we don't want to remap the zenith angle when there's an analytical exact formula to compute it exactly on each grid.

Reply:
@kmdeck could we add this to ClimaLand?